editor/editor_asset_installer.cpp

```diff
@@ -261,6 +262,14 @@ void EditorAssetInstaller::ok_pressed() {
 				da->make_dir(dirpath);
 				memdelete(da);
 
+				if (dirpath.begins_with("res://addons")) {
+					Vector<String> separated = dirpath.split("/");
+					String plugin_folder = separated[separated.find("addons") + 1];
+					if (plugin_folders.find(plugin_folder) == -1) {
```

Comment:
This is `O(n^2)`, but might be improved to `O(n log n)` by using `Set<String>` instead of `Vector<String>`. Probably even faster with `Set<StringName>`, as comparisons will be faster this way.

Reply:
I used a Set<StringName> instead of a Vector<String> now.

Reply:
No, I did not mean the `separated` variable; indeed, changing it into a Set would make it impossible to guess the plugin name.